### ISSUE TITLE
Self Reporting check box on New/Edit Skill dialog does not have a properly associated label

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1203]
The "Self Reporting" label for the checkbox is not associated via for=id nor does the checkbox have an aria-label, as such, screenreaders have no context for what that checkbox means, it's read as "checkbox checked/not checked". Associate the existing label using for=id or add aria-label to the checkbox.

**Comment by rmmayo** [comment]
PR: #1426

**Comment by mosefer** [comment]
:+1:
